### PR TITLE
limit parallelism to 10 for jax with foss/2020b, to avoid failing tests because not enough threads can be started

### PR DESCRIPTION
**Pull request by boegel**
@Flamefire for https://github.com/easybuilders/easybuild-easyconfigs/pull/13760, to avoid the failing tests I'm running into when using 36 cores for running the tests, cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/13760#issuecomment-907046372

### CODE REVIEW

**Comment by boegel** [comment]
I've added this to https://github.com/easybuilders/easybuild-easyconfigs/pull/13760, time to get this show on the road...

File: easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb [hunk 1]
```diff
@@ -34,6 +34,10 @@ local_jax_prebuildopts = "sed -i 's$pathToSed$%s$g' WORKSPACE && " % local_tf_bu
 
 use_pip = True
 
+# running the tests with lots of cores results in test failures because not enough threads can be started,
+# so limit to a reasonable number of cores to use at maximum
+maxparallel = 10
```
Comment (Flamefire) [inline easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb:39]: FTR: This needlessly limits the parallelism for all packages, e.g. jaxlib which now builds slower. We need this option for extensions...

Comment (boegel) [reply easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb:39]: I didn't think of that, but setting this only for the `jax `extension may actually work already... I'll check on that.

Comment (boegel) [reply easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb:39]: Doesn't work currently, but it's an easy fix, cfr. https://github.com/easybuilders/easybuild-framework/pull/3811

Comment (boegel) [reply easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb:39]: fixed in `jax` easyconfig in #13851

**Pull request merged**